lib/paymentrails/gateways/RecipientGateway.rb

```diff
@@ -25,10 +25,35 @@ def update(recipient_id, body)
     end
 
     def delete(recipient_id)
-      @client.delete('/v1/recipients/' + recipient_id)
+      path = '/v1/recipients/'
+      body = ''
+
+      if recipient_id.is_a?(String)
+        path += recipient_id
+      elsif recipient_id.is_a?(Array)
+        body = { ids: recipient_id }
+      else
+        raise 'Invalid recipient_id type'
+      end
+
+      @client.delete(path, body)
       true
     end
 
+    def find_logs(recipient_id)
```

Comment:
Please add a comment clarifying method's purpose

Reply:
added in https://github.com/PaymentRails/ruby-sdk/pull/52/commits/7e87f13388e14242686a9b87d65c5995d35a034f